src/controls/Switch.qml

```diff
@@ -23,6 +23,15 @@ import Material 0.3
 Controls.Switch {
     id: control
 
+    property Action action
```

Comment:
can you add a property that flags the display of checkable actions as switch. For example : property bool displayCheckableActionAsSwitch : false